operations/utility_storage/Troubleshoot_Ceph_New_RGW_Deployment_Failing.md

```diff
@@ -0,0 +1,149 @@
+# Troubleshoot Ceph New RGW Deployment Failing
+
+Troubleshoot an issue where a new RGW deployment is failing because of an address already in use. This bug corrupts the health of the Ceph cluster.
+
+Return the Ceph cluster to a healthy state by resolving issues with failing deployment.
+
+## Prerequisites
+
+This procedure requires admin privileges.
+
+## Procedure
+
+See [Collect Information about the Ceph Cluster](Collect_Information_About_the_Ceph_Cluster.md) for more information on how to interpret the output of the Ceph commands used in this procedure.
+
+### Verify Ceph is deploying a second RGW deployment
+
+1. Log on to a master node or `ncn-s001/2/3` to run the following commands.
+
+1. Check that the issue is occuring
+
+    1. Check the ceph admin tool
```

Comment:
```suggestion
    1. Use the `cephadm` logs to see if this error is occurring. The command below is following the `cephadm` logs. It may take a minute or two for the Ceph to attempt to deploy RGW and for the error to appear.
```